lib/restpack_serializer/serializable.rb

```diff
@@ -72,6 +73,24 @@ def add_custom_attributes(data)
       data.merge!(custom) if custom
     end
 
+    def apply_whitelist_and_blacklist(context)
+      if context[:attribute_blacklist].present?
+        self.class.serializable_attributes.each do |key, value|
+          if context[:attribute_blacklist].include? key
+            context[value[:include_method_name]] = false
+          end
+        end
+      end
+
+      if context[:attribute_whitelist].present?
+        self.class.serializable_attributes.each do |key, value|
+          unless context[:attribute_whitelist].include? key
+            context[value[:include_method_name]] = false
```

Comment:
This implies that if an attribute is in both lists the blacklist always takes precedence, which is ideal imho. Worth failing hard if an attribute is in both lists to eliminate surprise?